collectors/APICalls/TrackerTracker.js

```diff
@@ -74,37 +106,71 @@ class TrackerTracker {
                 throw new Error('API unavailable in given context.');
             }
 
+            // only save arguments if requested for given breakpoint
+            const argumentCollection = breakpoint.saveArguments ? `args: Array.from(arguments).map(a => a.toString())` : '';
+
             let conditionScript = `
-                const data = {
-                    description: '${description}',
-                    stack: (new Error()).stack
-                };
-            `;
+                const stack = (new Error()).stack;
```

Comment:
This is getting complex and is not linted, it'd be better to move it to a separate file / privacy function and use function.toString() here. We can leave it for later though, up to you.